docs/antora/ccutil/guides-operator/master.adoc

```diff
@@ -0,0 +1,7 @@
+:toc:
+:toclevels: 3
+:numbered:
+:linkattrs:
+
+include::guides-operator/20.0/guides-keycloak-operator-ccutil.adoc[]
```

Comment:
Should this use `{project_versionDoc}`?

Reply:
done, this is now using an unversioned Antora component, so it is no longer part of the path.